features/steps/test_post_endpoint.rb

```diff
@@ -3,13 +3,16 @@
 class Spinach::Features::TestPostEndpoint < Spinach::FeatureSteps
   before do
     @credentials_manager = CredentialsManager.new
+    p ENV["USERNAME"]
```

Comment:
Can you take these print statements out.

Reply:
removed